stable/traefik/values.yaml

```diff
@@ -408,6 +408,10 @@ metrics:
     ## it to the dashboard service only
     restrictAccess: false
     # buckets: [0.1,0.3,1.2,5]
+    serviceMonitor:
+      enabled: false
+      namespace: ""
```

Comment:
Is the `""`  equal to `nil`?

Reply:
Basically following the exact pattern of [nginx-ingress](https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L349-L353).

Reply:
👌